packages/a11y/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@kaizen/a11y",
+  "version": "1.0.0",
+  "description": "A component to hide elements visually, but still make the accessible to screen readers",
```

Comment:
This doesn't matter a huge amount but this package is likely to hold other accessibility helpers, whereas this description is just for visually hidden.